src/components/global/Playground/playground.css

```diff
@@ -213,6 +241,79 @@
   }
 }
 
+.playground__console {
```

Comment:
Everything works great! I think we should reduce the font size of the console however. It is larger than any other UI within the playground (including the code snippets). 

The code's font size is `.813rem`. Perhaps we start there to align visually?